components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/internal/impl/UserSessionManagementServiceImpl.java

```diff
@@ -57,17 +61,71 @@ public void terminateSessionsOfUser(String username, String userStoreDomain, Str
             UserSessionException {
 
         validate(username, userStoreDomain, tenantDomain);
-        List<String> sessionListOfUser = getSessionsOfUser(username, userStoreDomain, tenantDomain);
 
-        if (!sessionListOfUser.isEmpty()) {
-            if (log.isDebugEnabled()) {
-                log.debug("Terminating all the active sessions of user: " + username + " of userstore domain: " +
-                        userStoreDomain + " in tenant: " + tenantDomain);
+        String userId = resolveUserIdFromUsername(getTenantId(tenantDomain), userStoreDomain, username);
```

Comment:
With the change, we have removed the debug log that was previously there. Is there any specific reason to do that?

Reply:
Fixed 8a7f2322bf7980e1b678bc34667f82699d5d1ce1